src/util/rsmemory.h

```diff
@@ -146,6 +148,7 @@ template<typename T = void> rs_owner_ptr<T> rs_malloc(
 		{
 			RS_ERR( "A chunk of size larger than ", SAFE_MEMALLOC_THRESHOLD,
 			        " was requested" );
+			print_stacktrace();
 			exit(static_cast<int>(std::errc::argument_out_of_domain));
```

Comment:
The exit here is justified (contrary to the other ones), because allocating a large memory block is an error in the code. We could of course return null and let the caller take care of it.